<launch>
  <arg name="port_name"                default="/dev/ttyUSB0"/>
  <arg name="baud_rate"                default="460800"/>
  <arg name="ctl_rate_hz"                default="10"/>

  <param name="futaba_rs_info"          value="$(find futaba_rs_controller)/config/upperbody2.yaml"/>

  <node type="futaba_rs_controller" pkg="futaba_rs_controller" name="futaba_rs_controller" required="true" output="screen">
    <param name="port_name" value="$(arg port_name)" />
    <param name="baud_rate" value="$(arg baud_rate)" />
    <param name="ctl_rate_hz" value="$(arg ctl_rate_hz)" />

    <!-- 自分用 -->
	  <remap from="/futaba_rs_controller/goal_joint_states" to="joint_states"/> 

  </node>
</launch>
